### PR TITLE
Updated bundle manifest for 4.11

### PR DESCRIPTION
**Pull request by aneeshkp**
Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>
Updated the main manifest to 4.11 and ran make a bundle to generate manifests for 4.11 branch

### CODE REVIEW

File: bundle/manifests/ptp-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -334,7 +334,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.name
-                image: quay.io/openshift/origin-ptp-operator:4.10
+                image: quay.io/aneeshkp/ptp-operator
```
Comment (josephdrichard) [inline bundle/manifests/ptp-operator.clusterserviceversion.yaml:337]: upstream image?

File: config/manager/kustomization.yaml [hunk 2]
```diff
@@ -12,7 +12,6 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openshift/origin-ptp-operator
-  newTag: "4.10"
+  newName: quay.io/aneeshkp/ptp-operator
```
Comment (josephdrichard) [inline config/manager/kustomization.yaml:15]: upstream image?

File: bundle/manifests/ptp-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (aneeshkp) [reply bundle/manifests/ptp-operator.clusterserviceversion.yaml]: fixed

File: config/manager/kustomization.yaml (hunk 2 above)
Comment (aneeshkp) [reply config/manager/kustomization.yaml]: fixed

**Pull request merged**